src/cljs/rems/application.cljs

```diff
@@ -411,6 +411,7 @@
     "date" [date-field f]
     "attachment" [attachment-field f]
     "label" [label f]
+    "description" [text-field f]
```

Comment:
the function of this itemtype is probably not obvious to everyone so maybe document the purpose somewhere